lib/config/optionsconfiguration.go

```diff
@@ -143,6 +143,7 @@ type OptionsConfiguration struct {
 	KCPSendWindowSize       int                     `xml:"kcpSendWindowSize" json:"kcpSendWindowSize" default:"128"`
 	KCPReceiveWindowSize    int                     `xml:"kcpReceiveWindowSize" json:"kcpReceiveWindowSize" default:"128"`
 	DefaultFolderPath       string                  `xml:"defaultFolderPath" json:"defaultFolderPath" default:"~"`
+	SetLowPriority          bool                    `xml:"setLowPriority" json:"setLowPriority" default:"true"`
```

Comment:
I'd make this an int, 0 means normal, 1 means BE towards the end, 2 means IDLE etc.

Reply:
Meh, do not want. I think we should just provide a reasonable default and let the user do the setup themselves if they care more than that. The only reason for a config at all is to have an off switch to avoid screwing with the potential user settings...